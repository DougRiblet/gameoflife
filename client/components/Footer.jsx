import React from 'react'

export default class Footer extends React.Component {
  render () {
    return (
      <div>
        <p>
          Created by Doug Riblet | <a href='https://github.com/smashzen'>github.com/smashzen</a>
        </p>
      </div>
    )
  }
}
