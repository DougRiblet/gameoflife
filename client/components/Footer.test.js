import React from 'react'
import ReactDOM from 'react-dom'
import Footer from './Footer'
import renderer from 'react-test-renderer'

test('Footer component displays footer', () => {
  const component = renderer.create(
    <Footer />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

describe('Footer', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<Footer/>, div)
    })
})
