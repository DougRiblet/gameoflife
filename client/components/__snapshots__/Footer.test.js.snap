// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component displays footer 1`] = `
<div>
  <p>
    Created by Doug Riblet | 
    <a
      href="https://github.com/smashzen"
    >
      github.com/smashzen
    </a>
  </p>
</div>
`;
